doc/source/release-notes/Percona-Server-8.0.27-18.rst

```diff
@@ -0,0 +1,70 @@
+`MySQL 8.0.27 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-27.html>`__.
+in addition to enterprise-grade features developed by Percona.
+
+*Percona Server for MySQL* is a free, fully compatible, enhanced and open
+source drop-in replacement for any *MySQL* database. It provides superior
+performance, scalability and instrumentation.
+
+*Percona Server for MySQL* is trusted by thousands of enterprises to provide
+better performance and concurrency for their most demanding workloads, and
+delivers greater value to *MySQL* server users with optimized performance,
+greater performance scalability and availability, enhanced backups and
+increased visibility. `Commercial support contracts are available
+<https://www.percona.com/services/support/mysql-support>`__.
+
+Release Highlights
+=================================================
+
+The following list are some of the bug fixes for *MySQL* 8.0.27, provided by Oracle, and included in Percona Server for MySQL:
+
+* The ``default_authentication_plugin`` is deprecated. Support for this plugin may be removed in future versions. Use the ``authentication_policy`` variable.
+* The ``binary`` operator is deprecated. Support for this operator may be removed in future versions. Use ``CAST(... AS BINARY)``.
+* Fix for when a parent table initiates a cascading ``SET NULL`` operation on the child table, the virtual column can be set to NULL instead of the value derived from the parent table.
+
+
+Find the full list of bug fixes and changes in the `MySQL 8.0.27 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-27.html>`__.
+
+
+New Features
+================================================================================
+
+* :jirabug:`PS-7960`: Document the ``rocksdb_partial_index_sort_max_mem`` variable, the ``rocksdb_bulk_load_partial_index`` variable, and the ``rocksdb_cancel_manual_compactions`` variable.
+
+
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-7955`: Enable ZenFS functionality on standard Percona Server packages on Debian 11 and Ubuntu 20.04.
+* :jirabug:`PS-7931`: Implement a Slow Query Log rotation in Percona Server for MySQL 8.0.
+* :jirabug:`PS-6730`: The 'Last_errno:' field in the Slow Query Log contains only error information.
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-7883`: An ``ALTER`` query causes a server exit when ``--rocksdb_write_disable_wal`` is enabled.
+* :jirabug:`PS-8007`: *Percona Server for MySQL* can fail to start if the server starts before the network mounts the datadir or local mount of the datadir.
+* :jirabug:`PS-7977`: The AppArmor profile is broken after an 8.0.22-13 to 8.0.23-14 upgrade. (Thanks to Alex Kompel for reporting this issue)
+* :jirabug:`PS-2346`: LP #720547: Implement an option to allow queries with specific error codes to add entries to the Slow Query Log.
```

Comment:
Thas looks more like a new feature or improvement to me?

Reply:
Confirmed with Oleksander - this is a new feature.

Reply:
Moved to new features.

Reply:
Correct, the JIRA issue type was incorrect.